### PR TITLE
agg, fix download/build/libpng16 reference

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/agg/agg-2.5.recipe [hunk 1]
```diff
@@ -36,71 +34,80 @@ PROVIDES_devel="
...
 	lib:libaggplatformsdl$secondaryArchSuffix = 2.0.4 compat >= 2
-"
+	"
 REQUIRES_sdl="
 	agg$secondaryArchSuffix == $portVersion base
 	haiku$secondaryArchSuffix
-	lib:libGL$secondaryArchSuffix
-	lib:libsdl_1.2$secondaryArchSuffix
-"
+	lib:libSDL$secondaryArchSuffix
```
Comment (korli) [inline x11-libs/agg/agg-2.5.recipe]: This was more correct before.

Comment (Begasus) [reply x11-libs/agg/agg-2.5.recipe]: Shouldn't it be like libSDL for 1.* and libSDL2_2.0 to declare them?

Comment (Begasus) [reply x11-libs/agg/agg-2.5.recipe]: Searching on haikuports it seems that 185 recipes(?) are using lib:libsdl and 39 are using lib:libsdl_1.2, if things need to be changed it's better to get an agreement on this ;)

**Comment by pulkomandy** [comment]
On Sun, May 19, 2019 at 10:34:03AM -0700, Schrijvers Luc wrote:
> Begasus commented on this pull request.
> 
> 
> 
> >  REQUIRES_sdl="
>  	agg$secondaryArchSuffix == $portVersion base
>  	haiku$secondaryArchSuffix
> -	lib:libGL$secondaryArchSuffix
> -	lib:libsdl_1.2$secondaryArchSuffix
> -"
> +	lib:libSDL$secondaryArchSuffix
> 
> Searching on haikuports it seems that 185 recipes(?) are using lib:libsdl and 39 are using lib:libsdl_1.2, if things need to be changed it's better to get an agreement on this ;)

lib:libsdl is not correct. The package does not contain a
libsdl.so.something, only a libsd-1.2.so.something.

So all users of lib:libsdl should be fixed to use lib:libsdl_1.2, and
then the lib:libsdl provide removed from the SDL package.

**Comment by Begasus** [comment]
> On Sun, May 19, 2019 at 10:34:03AM -0700, Schrijvers Luc wrote: Begasus commented on this pull request. > REQUIRES_sdl=" agg$secondaryArchSuffix == $portVersion base haiku$secondaryArchSuffix - lib:libGL$secondaryArchSuffix - lib:libsdl_1.2$secondaryArchSuffix -" + lib:libSDL$secondaryArchSuffix Searching on haikuports it seems that 185 recipes(?) are using lib:libsdl and 39 are using lib:libsdl_1.2, if things need to be changed it's better to get an agreement on this ;)
> lib:libsdl is not correct. The package does not contain a libsdl.so.something, only a libsd-1.2.so.something. So all users of lib:libsdl should be fixed to use lib:libsdl_1.2, and then the lib:libsdl provide removed from the SDL package.

Ok, so another issue like libpng? Will fix this as soon as I'm able for this.

**Pull request merged**